buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -17,3 +17,9 @@ fun stately(
     module: String,
     version: String = "1.1.1-a1"
 ): String = "co.touchlab:stately-$module:$version"
+
+object androidx {
+    fun startup(
+        version: String = "1.0.0"
+    ) = "androidx.startup:startup-runtime:$version"
+}
```

Comment:
❤️

Reply:
Loved that this allowed me to remove `Context` from the API (so all primary operations can be defined in `common`). A clean/simple/unified API. ❤️